cli/src/config.rs

```diff
@@ -50,6 +50,7 @@ pub struct TypeScriptParams {
 pub struct GoParams {
     pub package: String,
     pub uppercase_acronyms: Vec<String>,
+    pub no_pointer_slice: bool,
```

Comment:
The description of the problem in the PR was very good. Could we copy/paste some of that prose here as a doc comment

Reply:
Done!